### PR TITLE
improvement: various add-ons - stop using printStackTrace

### PR DESCRIPTION
**Pull request by kingthorin**
- accessControl > Replaced two prints with debug logging. Fixed a typo in the CHANGELOG and added maintenance entry.
- domxss > Replaced one print with debug logging. CHANGELOG added maintenance entry.
- pscanrulesAlpha > Removed one commented print.
- scripts > Replaced one print with debug logging. (Unreleased section already has maint note.)
- zest > Replaced three prints with debug logging. CHANGELOG added maintenance entry.

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: addOns/accessControl/src/main/java/org/zaproxy/zap/extension/accessControl/AccessControlAlertsProcessor.java [hunk 1]
```diff
@@ -88,7 +88,7 @@ private void raiseAuthorizationAlert(AccessControlResultEntry result) {
         try {
             msg = result.getHistoryReference().getHttpMessage();
         } catch (HttpMalformedHeaderException | DatabaseException e) {
-            e.printStackTrace();
+            log.debug(e.getMessage(), e);
```
Comment (thc202) [inline addOns/accessControl/src/main/java/org/zaproxy/zap/extension/accessControl/AccessControlAlertsProcessor.java:91]: Could just pass `e`, same in other places.

File: addOns/accessControl/CHANGELOG.md [hunk 2]
```diff
@@ -6,7 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## Unreleased
 ### Changed
 - Don't set the font color for inherited entries (Issue 6397).
-- Upate minimum ZAP version to 2.10.0.
+- Update minimum ZAP version to 2.10.0.
```
Comment (thc202) [inline addOns/accessControl/CHANGELOG.md:9]: Good catch.

File: addOns/tlsdebug/src/main/java/org/zaproxy/zap/extension/tlsdebug/HttpsCallerProcess.java [hunk 3]
```diff
@@ -55,7 +58,7 @@ public static void main(String[] args) {
             }
             System.out.println("--- END of call -------------------------------------------------");
         } catch (MalformedURLException e) {
-            e.printStackTrace();
```
Comment (thc202) [inline addOns/tlsdebug/src/main/java/org/zaproxy/zap/extension/tlsdebug/HttpsCallerProcess.java:58]: This is actually correct, this is a process spawn by the add-on which consumes its output.

**Comment by kingthorin** [comment]
Tweaked

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Pull request merged**